.gitmodules

```diff
@@ -9,3 +9,5 @@
 [submodule "Node-Externals"]
 	path = src/submodules/Node-Externals
 	url = https://github.com/dotnet/Node-Externals
+	branch = main
+	depth = 1
```

Comment:
We don't need to clone the entire history for this repo, a bare clone is lighter and faster